job-opportunities/internships/README.md

```diff
@@ -1,2 +1,7 @@
 # Internships
 
+```{toctree}
+labeling-medical-images-for-a.i.-applications
+medical-image-analysis
+manual-correction-of-spinal-cord-segmentation-on-mri-data
```

Comment:
I assume this is out of scope for this PR, but has this page always been blank?

https://wiki-demo.neuropoly.org/job-opportunities/internships/manual-correction-of-spinal-cord-segmentation-on-mri-data.html

Reply:
Fixed (on `master`): 5d609ad